### PR TITLE
[Turnstile] 401 error on local development

### PR DESCRIPTION
**Pull request by patriciasantaana**
PCX-7965
https://community.cloudflare.com/t/allow-localhost-or-127-0-0-1-as-acceptable-domains-for-turnstile/423897

### CODE REVIEW

File: content/turnstile/frequently-asked-questions.md [hunk 1]
```diff
@@ -219,6 +219,15 @@ If you encounter an endless challenge loop, try disabling your browser extension
 {{</faq-answer>}}
 {{</faq-item>}}
 
+{{<faq-item>}}
+{{<faq-question level=2 text="What if I receive a 401 network error when developing locally?" >}}
+{{<faq-answer>}}
+
+Cloudflare returns a 401 status as part of the Private Access Token (PAT) challenge on `…/pat/…` URLs. You can use Turnstile on `127.0.0.1` or `localhost` as a domain to allow for local development.
```
Comment (punkeel) [inline content/turnstile/frequently-asked-questions.md:226]: I'm not sure this is accurate, the linked post is a bit confusing.

401 happens everywhere and is PAT.

If you want to test locally you need to allow 127.0.0.1 / localhost, or you'll get errors (along the lines of "This domain is not allowed")

These two problems are not really related 😅 


We already have a Q for local mode: `Can I use Turnstile when developing locally?`

Comment (patriciasantaana) [reply content/turnstile/frequently-asked-questions.md:226]: thanks for checking it out! I just took the answers from the community post and added it here. I suppose we can close this out
